### DISCUSSION TITLE
Can Flux reconcilation is active only in a time period

### DISCUSSION THREAD

**Discussion by eugercek** [fluxcd/flux2#3341] [Q&A] [answered]
Hi all, 

I need to run flux reconciliation only in a period of time how can I achieve this?

The problem with interval is, it checks the source with the interval but I need to make reconciliation happen in a fixed period of time with an unimportant interval.

If no there is no built-in support do you have any advice to achieve this

Thanks :pray:

**Comment by stefanprodan** [comment, marked as answer]
You can do `flux suspend source git` and Flux will not apply new changes made in Git, to resume do `flux resume source git`. The `suspend/resume` commands are available for all sources, Flux Kustomizations and HelmReleases. 

In the future we plan to support time-based gating, please see https://github.com/fluxcd/flux2/pull/3158

**Reply by eugercek** [reply]
Cool RFC. I hope it'll be implemented (+1 for closing gates (freeze) for sources instead of objects). And thanks for your advice, I planned to do it as you said, but this process needs to be automated. It's not for wait before prod-deployment like situation. I'll probably use linux cronjob, does anyone have better idea?
